staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -390,9 +429,14 @@ func (crEncoder) Encode(obj runtime.Object, w io.Writer) error {
 	}
 }
 
-type UnstructuredCreator struct{}
+type unstructuredCreator struct {
+	delegate runtime.ObjectCreater
+}
 
-func (UnstructuredCreator) New(kind schema.GroupVersionKind) (runtime.Object, error) {
+func (c unstructuredCreator) New(kind schema.GroupVersionKind) (runtime.Object, error) {
+	if obj, err := c.delegate.New(kind); err == nil {
```

Comment:
Doesn't this mean that certain types become invalid targets for CRDS?  `ListOptions`, `ExportOptions`, etc?  Not that I object, I just want to be sure I understand the limitation.